src/_product-updates/2025-07-25-embed-mode.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Introducing Embed mode 
+tags: [New]
+image: /images/changelog/embed-mode.png
+---
+
+![embed-mode.png](/images/changelog/embed-mode.png)
+
+Thanks to Embed mode, documentation can now be fully integrated in your global navigation, to provide your users with one unified experience. 
+By integrating your Bump.sh documentation inside your own frontend stack, you get complete control over your branding and navigation.
+
+To activate Embed mode, three steps are needed:
```

Comment:
Wouldn't it be worth mentioning in the release note that an Enterprise plan is required?